### PR TITLE
[20180608-sev-flakes] Extend socket timeout (after socket was created...)

### PR DESCRIPTION
**Pull request by juleea**

### CODE REVIEW

File: lib/gusto-karma.config.js [hunk 1]
```diff
@@ -127,10 +127,6 @@ module.exports = function(config) {
     singleRun: true,
     plugins: plugins,
     reporters: reporters,
-    mochaReporter: {
-      showDiff: true,
-      output: 'minimal'
-    },
```
Comment (kurtzur) [inline lib/gusto-karma.config.js]: why did we remove this?

**Comment by kurtzur** [comment]
[this issue](https://stackoverflow.com/questions/12815231/controlling-the-heartbeat-timeout-from-the-client-in-socket-io) is what led me to believe that changing the timeout after instantiation would work

**Review by kurtzur — APPROVED** [review]

**Comment by zp-alfred** [comment]
Accepted by kurtzur!

**Comment by zp-alfred** [comment]
Accepted by kurtzur!

**Pull request merged**